src/browser.ts

```diff
@@ -51,9 +51,9 @@ export function activateOpenInBrowser(
     execute: () => {
       const file = browser.selectedItems().next();
```

Comment:
I cannot open the HDF5 file by double-clicking in the browser :disappointed: ...

It tells me that `browser` is `undefined` here. Is it working fine for you ?

Reply:
Are you sure that you are on JupyterLab 4? I guess you are using JupyterLab 3 and that's where the `browser` is `undefined` error comes from.

Reply:
Pretty sure I was using JLab4. I'll have a second go.

Reply:
![Screenshot_20230628_104103](https://github.com/silx-kit/jupyterlab-h5web/assets/44365948/539cae53-581f-45fb-9a8a-112a5ff1a19e)

I started in a clean env and I could open the files.

Reply:
Yeah, you were right: the problem was on my end. I was indeed on JLab4 but had probably some stale builds lying around.

I could make it work locally and on Binder :+1: